MapboxNavigation.podspec

```
@@ -44,7 +44,7 @@ Pod::Spec.new do |s|
   s.module_name = "MapboxNavigation"
 
   s.dependency "MapboxCoreNavigation", "#{s.version.to_s}"
-  s.dependency "MapboxMaps", "~> 10.0"
+  s.dependency "MapboxMaps", "~> 10.1"
```

Is it better to use this syntax?
```suggestion
  s.dependency "MapboxMaps", ">= 10.1.1", "< 11.0.0"
```